src/components/Crypto/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {CryptoTypes} from '../../types';
+import {
+  CryptoContainer,
+  NameContainer,
+  Price,
+  Logo,
+  Increase,
+  Decrease,
+  Name,
+  Slug,
+  Arrow,
+  Arrowdown,
+  Arrowup,
+  Percentage,
+} from '../Crypto/styles';
+
+const Crypto = ({name, slug, price, market_porcentage, image}: CryptoTypes) => {
+  return (
+    <CryptoContainer>
+      <Logo source={image} />
+      <NameContainer>
+        <Name>{name}</Name>
+        <Slug>{slug}</Slug>
+      </NameContainer>
+      <Price>${price}</Price>
+      {parseInt(market_porcentage) > 0.0 ? (
```

Comment:
use > 0  in the validation

Also you can send via prop the boolean, and change the color in the component, to prevent code redundance. it should be the same component with a validation inside the state to change the color